### ISSUE TITLE
NVDA cannot operate text selection in browse mode when using "Default NVDA word navigation"

### ISSUE THREAD

**Issue by cary-rowen** [mltony/nvda-word-nav#13]
## Steps to reproduce
1. Assign left Ctrl to "Default NVDA word navigation (WordNav disabled)".
2. Open web page and try to select text by pressing Shift+Ctrl+RightArrow in browse mode.
## actual behavior
NVDA does not report that the "selected", and pressing Ctrl+C also reports that the text is not selected.

Even if "Enable WordNav in browse mode" is unchecked in the settings panel.

**Comment by cary-rowen** [comment]
cc @mltony 
I added steps to reproduce, I'm using the latest master build

**Comment by mltony** [comment]
Just pushed a fix - could you try master branch?

**Comment by cary-rowen** [comment]
Yes This has been fixed
